### ISSUE TITLE
[Larger issue] Documentation lacks detailed API specifications for features

### ISSUE THREAD

**Issue by desa** [influxdata/docs.influxdata.com#612]
## TL;DR

We need to have an API specification so that people can build against our product. 
## Issue

Discovered this when I was going through the CQ docs.

The section that brought this to my attention was the following:

```
...
 A successful `CREATE CONTINUOUS QUERY` statement returns an empty response.
If you attempt to create a continuous query that already exists, InfluxDB does not return an error.
...
```

This explanation is great, but operates on the assumption that the reader is working with the CLI and wouldn't be sufficient if the reader was building against the API. In particular, it has no mention of HTTP status codes or the expected format of the response.

This is an issue not just for this particular section, but for the company as a whole.
###### URL for relevant page?

https://github.com/influxdata/docs.influxdata.com/blob/master/content/influxdb/v1.0/query_language/continuous_queries.md#the-create-continuous-query-statement
